src/hotspot/share/gc/g1/g1CardSetContainers.inline.hpp

```diff
@@ -257,23 +257,42 @@ inline size_t G1CardSetBitMap::header_size_in_bytes() {
     return offset_of(G1CardSetBitMap, _bits);
 }
 
+inline G1CardSetHowl::ContainerPtr const* G1CardSetHowl::container_addr(EntryCountType index) const {
+  assert(index < _num_entries, "precondition");
+  return buckets() + index;
+}
+
+inline G1CardSetHowl::ContainerPtr* G1CardSetHowl::container_addr(EntryCountType index) {
+  assert(index < _num_entries, "precondition");
+  return const_cast<ContainerPtr*>(buckets() + index);
```

Comment:
While the implementation is very short here, one could chose to implement one in terms of the other.
```suggestion
  return const_cast<ContainerPtr*>(const_cast<const G1CardSetHowl*>(this)->container_addr(index));
```